src/variables/variable_unit_flow.jl

```diff
@@ -35,18 +35,21 @@ function unit_flow_indices(
     direction=anything,
     stochastic_scenario=anything,
     t=anything,
-)
+    temporal_block=temporal_block(ordering_periods=nothing)  #this is the addition for each variable to be made
+    )
     unit = members(unit)
     node = members(node)
+    unique(
     [
         (unit=u, node=n, direction=d, stochastic_scenario=s, t=t)
         for
-        (u, n, d, tb) in
-        unit__node__direction__temporal_block(unit=unit, node=node, direction=direction, _compact=false)
+        (u, n, d, temporal_block) in
+        unit__node__direction__temporal_block(unit=unit, node=node, direction=direction,temporal_block=temporal_block,_compact=false)
         for
         (n, s, t) in
-        node_stochastic_time_indices(m; node=n, stochastic_scenario=stochastic_scenario, temporal_block=tb, t=t)
+        node_stochastic_time_indices(m; node=n, stochastic_scenario=stochastic_scenario, temporal_block=temporal_block, t=t)
```

Comment:
Why this change? I think it's more consistent to name the iteration variable `tb`, as it was before...